packages/flutter_tools/test/integration.shard/analyze_once_test.dart

```diff
@@ -100,12 +100,27 @@ void main() {
     );
   });
 
-  // testWithoutContext a specific file outside the current directory
-  testWithoutContext('passing one file throws', () async {
+  testWithoutContext('passing one file works', () async {
```

Comment:
These tests look good, can you add two more test cases: `'passing one file with errors are detected'` and `'passing more than one file without errors works'`?

Reply:
Sure